ansible/roles/products/hand_e/docker-container/defaults/main.yml

```diff
@@ -1,11 +1,13 @@
 ---
 container_name: "dexterous_hand_real_hw"
-tag: "kinetic-release"
-image: "shadowrobot/dexterous-hand"
+image: "shadowrobot/dexterous-hand:kinetic-release"
 ethercat_interface: "eth0"
 launch_hand: true
 config_branch: demohand_A
 use_cyberglove: false
 user_folder: "/home/{{ lookup('env','USER') }}"
 shadow_hand_launcher_folder: "{{ user_folder }}/.shadow_launcher_app/shadow_hand_launcher"
-cyberglove: "A-dg"
\ No newline at end of file
+cyberglove: "A-dg"
+reinstall: false
+container_exists: false
```

Comment:
are these input data or variables which are defined during execution of the role?

Reply:
They are default values and they are defined here. The values are changed during execution of the role, but the definition happens in defaults for container_exists and container_running